### DISCUSSION TITLE
Bulk change images path

### DISCUSSION THREAD

**Discussion by engdarkalchemist** [freeplane/freeplane#1582] [General]
Hi team,
Recently I have changed my freeplane mind map (full folder) to another folder. Now, all images appear as `'FileNotFoundException' ... (The system cannot find the path specified) file:///C:/Users/..."`
How can bulk change the images path instead of change one by one?
Is it possible change path to relative path instead of absolute path for images? If so, please provide a guidance. (Note: I have set "relative" as 'Hyperlinks types'.)
Thank you so much!

**Comment by sifran-github** [comment]
One easy way is to open the mind map in a text editor (notepad) and do search & replace.
If you know what you are doing, its very easy. If you don't, you can ruin the mind map. so always backup before.

**Comment by engdarkalchemist** [comment]
Hi [sifran-github](https://github.com/sifran-github)
Thank you. I use notepad++. I thought this but I would want to know if there is a direct way through Freeplane. If you find, please, tell me.
Is there any way to set the images as relative path like in 'Hyperlinks types'?
